cmd/cli/plugin.go

```diff
@@ -24,10 +24,10 @@ func pluginCommand(plugins func() discovery.Plugins) *cobra.Command {
 			}
 
 			if !quiet {
-				fmt.Printf("%-20s\t%-s\n", "NAME", "LISTEN")
+				fmt.Printf("%-20s\t%-8s\t%-s\n", "NAME", "PROTOCOL", "LISTEN")
 			}
 			for k, v := range entries {
-				fmt.Printf("%-20s\t%-s\n", k, v.String())
+				fmt.Printf("%-20s\t%-8s\t%-s\n", k, v.Protocol, v.Address)
```

Comment:
Seems like name and address (path) are the only details relevant to users for now...so we should nix `v.Protocol`.

We can (and should) surface this when we support it in a usable form.